### ISSUE TITLE
Flash cards are generated within document to practice

### ISSUE THREAD

**Issue by ushabuntu** [remnoteio/remnote-issues#634]
**Describe the bug**
Finished editing a document with :: and cloze + images, but flash cards aren't generating. 
When attempting to practice the specific document (practice rem in this document with/without spaced reps) flash cards do not appear, saying I have no flash cards to practice. 

**Desktop (please complete the following information):**
 - OS: [MacOS]
 - Browser [Native app]
 - Version [1.1.6]

**Smartphone (please complete the following information):**
 - Device: [Galaxy s10e/Tab s6 Lite]
 - OS: [Android]
 - Browser [App]
 - Version [0.154]

**Additional context**
Flash cards aren't generating as expected.

**Comment by mfranzs** [comment]
Thanks for reaching out. Can you please elaborate on what you mean?

**Comment by ushabuntu** [comment]
Hey Martin!

Apologies, but the issue quite resolved by itself, so I figured I was just
drafting an issues, but not posting it..

In any case, I noticed that my document (with all the flash cards :: and
clozes with images) didn't generate the flash cards as expected, and just
told me I finished practicing.
I restarted the app, and it was fixed. Same on mobile.

It still occurs from time to time, but 5 minutes (give or take) lets the
flash cards generate.

Thanks for the support!

--
Uri
